From 3324c02f2b2f8be1354b99a6617aab7f2bc31df1 Mon Sep 17 00:00:00 2001
From: Dwane Pottratz <dwane.pottratz@intel.com>
Date: Mon, 11 Apr 2016 11:22:04 -0700
Subject: [PATCH 7/9] x86: config: enable NOCB_CPU and NOCB_CPU_ALL

RCU processing can induce some high latency events.  To avoid these we
enable NOCB_CPU and NOCB_CPU_ALL

Change-Id: Iac410fc851adf216dfd912cea1c93c7027b51555
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-28685
Signed-off-by: Dwane Pottratz <dwane.pottratz@intel.com>
Reviewed-on: https://android.intel.com:443/493210
---
 arch/x86/configs/x86_64_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index 3cbcfd0..76db671 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -138,7 +138,10 @@ CONFIG_RCU_FANOUT_LEAF=16
 CONFIG_RCU_FAST_NO_HZ=y
 # CONFIG_TREE_RCU_TRACE is not set
 # CONFIG_RCU_BOOST is not set
-# CONFIG_RCU_NOCB_CPU is not set
+CONFIG_RCU_NOCB_CPU=y
+# CONFIG_RCU_NOCB_CPU_NONE is not set
+# CONFIG_RCU_NOCB_CPU_ZERO is not set
+CONFIG_RCU_NOCB_CPU_ALL=y
 # CONFIG_IKCONFIG is not set
 CONFIG_LOG_BUF_SHIFT=18
 CONFIG_HAVE_UNSTABLE_SCHED_CLOCK=y
-- 
1.9.1

